unlock-app/.storybook/config.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import { configure, addDecorator } from '@storybook/react'
+import 'storybook-chromatic'
 import StoryRouter from 'storybook-react-router'
+
```

Comment:
nit: whitespace?

Reply:
I have gotten in the habit of putting a space between node_modules and internal imports. I'll remove this